tutorials/3d/portals/editing_rooms_and_portals.rst

```diff
@@ -21,46 +21,83 @@ Putting all the ideas together, here is an example scene tree:
 
-Rooms should be created as Empty objects with names such as ``Room_kitchen`` (substituting your preferred room name). If you place meshes within the rooms within the modeling tool, these Empties will form Spatial nodes in Godot, and will be converted to room nodes automatically during the room conversion phase.
+Postfix convention:
+^^^^^^^^^^^^^^^^^^^
 
-Creating a RoomGroup is done by creating an Empty object with the name prefix ``RoomGroup``. For instance, you can create an Empty object called ``RoomGroup_house`` and it will be converted to a RoomGroup.
+- ``-room`` becomes a :ref:`Room<class_Room>`.
+- ``-roomgroup`` becomes a :ref:`RoomGroup<class_RoomGroup>`.
+- ``-portal`` becomes a :ref:`Portal<class_Portal>`.
+- ``-bound`` becomes a manual bound.
 
-Creating Portals in modeling tools
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Rooms and RoomGroups should be created as Empties within Blender. Any mesh children of the ``-room`` Empty will thus be placed in the Room during conversion in Godot.
 
-Portals also use the same naming convention as within Godot - the name should start with the prefix ``Portal_`` followed by the name of the room the portal should link to. However, for portals, we want to specify the shape and position of the polygon within the modeling tool, so the nodes should be Meshes rather than Empties.
+e.g.
+
+- ``kitchen-room`` - create a Room called "kitchen".
+- ``outside-roomgroup`` - create a RoomGroup called "outside".
+- ``kitchen-portal`` - create a Portal leading to the "kitchen" Room.
+
+Portals
+^^^^^^^
+
+Portals are a little different than Rooms, in that we want to specify the geometry of the Portal in our modelling tool, in addition to just the name. To do this your "portal-to-be" should be created as a Mesh.
 
 Portal meshes have some restrictions to work properly. They must be convex, and the polygon points should be in the same plane. The accuracy to the plane does not have to be exact, as Godot will automatically average the direction of the portal plane. Once converted to a :ref:`Portal<class_Portal>` node, the snapping to the portal plane is enforced, and the vertices are specified (and editable) as 2D coordinates in the inspector, rather than 3D points.
 
+The naming of the portal is quite important. You can either name the portal ``-portal``, which will attempt to autolink the Portal in Godot, or you can use as a prefix the name of the Room which you wish it to link to.
+
+Wildcard
+^^^^^^^^
+
+In most cases this should be quite simple, using a name such as ``kitchen-portal``. However there is one problem - Blender, and Godot, do not deal well when you have multiple objects with the same name. What happens when we want more than one Portal to lead to the kitchen?
```

Comment:
```suggestion
In most cases, this can be done using a name such as ``kitchen-portal``. However, there is one problem. Blender and Godot do not deal well when you have multiple objects with the same name. What happens when we want more than one Portal to lead to the kitchen?
```